app/Http/Controllers/AccountManagementController.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use \Illuminate\Validation\ValidationException;
+use App\Services\AdminAccountManagementService;
+
+class AccountManagementController extends Controller
+{
+    /**
+     * Constructor
+     */
+    public function __construct()
+    {
+    }
+
+    /**
+     * 管理者アカウントの一覧取得 (ownerは自分の店舗に所属しているアカウントのみ取得可能)
+     * @return \Illuminate\Http\JSONResponse
+     */
+    public function adminList()
+    {
+        try {
+            $admins = AdminAccountManagementService::getAdminList();
+
+            // アカウント毎のrestaurant_idとrestaurant_nameを取得し、レスポンスに追加
+            foreach ($admins as $admin) {
+                $restaurants = AdminAccountManagementService::getRelatedRestaurant($admin->admin_id);
+                $admin['restaurant_id'] = array_column($restaurants, 'restaurant_id');
+                $admin['restaurant_name'] = array_column($restaurants, 'restaurant_name');
+            }
+        } catch (\Exception $e) {
+            return response()->json(['error' => $e->getMessage()], 500);
+        }
+
+        return response()->json($admins, 200);
+    }
+
+    /**
+     * 新規管理者アカウント(owner, counter, kitchen)発行
+     * @param Request $request リクエストボディ
+     * @return \Illuminate\Http\JSONResponse
+     */
+    public function adminSignUp(Request $request)
+    {
+        // リクエストボディのバリデーション (400エラーを返す)
+        try {
+            $request->validate([
+                'admin_name' => 'required|string',
+                'login_id' => 'required|string|unique:admins,login_id',
+                'password' => 'required|string',
+                'admin_role' => 'required|string|in:system,owner,counter,kitchen',
+                'restaurant_id' => 'array',
+                'restaurant_id.*' => 'integer',
+            ]);
+        } catch (ValidationException $e) {
+            return response()->json(['error' => 'リクエストの形式または内容に誤りがある'], 400);
+        }
+
+        $adminInfo = json_decode($request->getContent(), true);
```

Comment:
```suggestion
        $adminInfo = $request->validated();
```
[MAY] **laravel**
これでバリデーション済みのデータが取得できたはずです。

Reply:
`$adminInfo = $request->validated()`に変更すると、以下のエラーが出ます：
`BadMethodCallException: Method Illuminate\Http\Request::validated does not exist.`

Reply:
FormRequestにしか使えないそうです。